<ui version="4.0" >
 <class>advWidget</class>
 <widget class="QWidget" name="advWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QComboBox" name="ipcombo" >
     <item>
      <property name="text" >
       <string>Use Dynamic IPs</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Use Static IPs</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>IP</string>
          </property>
          <property name="buddy" >
           <cstring>ip</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="ip" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Netmask</string>
          </property>
          <property name="buddy" >
           <cstring>netmask</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="netmask" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Gateway</string>
          </property>
          <property name="buddy" >
           <cstring>gateway</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="gateway" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="dnscombo" >
     <item>
      <property name="text" >
       <string>Use Dynamic DNS</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Use Static DNS</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Use Global DNS</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Use OpenDNS</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>DNS1</string>
          </property>
          <property name="buddy" >
           <cstring>dns1</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="dns1" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>DNS2</string>
          </property>
          <property name="buddy" >
           <cstring>dns2</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="dns2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>DNS3</string>
          </property>
          <property name="buddy" >
           <cstring>dns3</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="dns3" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
